pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/errorprone/ApexCSRFRule.java

```diff
@@ -54,7 +68,7 @@ private void checkForCSRF(ASTMethod node, Object data) {
         }
 
         String name = node.getImage();
-        if (name.equalsIgnoreCase(INIT)) {
+        if (INIT.equalsIgnoreCase(name) || STATIC_INITIALIZER.equals(name)) {
```

Comment:
I'm not a java expert but I would combine those two ifs into one. If it gets unreadable then I would extract the outer two tests into a method